src/App/pages/RecipeDetails/RecipeDetails.tsx

```diff
@@ -78,24 +77,26 @@ const RecipeDetails = () => {
           <div className={styles.recipe__section}>
             <h2>Equipment</h2>
             <ul>
-              {recipe.analyzedInstructions[0]?.steps.map((step) => (
+              {recipeDetailsStore.recipe.analyzedInstructions[0]?.steps.map((step) => (
                 <li key={step.number}>{step.equipment.map((item) => item.name).join(', ')}</li>
               ))}
```

Comment:
[+] тут в нескольких местах похожим образом достается поле `steps`, можно эту логику унести в геттер в `RecipeDetailsStore`

```suggestion
              {recipeDetailsStore.steps.map((step) => (
                <li key={step.number}>{step.equipment.map((item) => item.name).join(', ')}</li>
              ))}
```

\+ в этом списке тоже лучше не рендерить лишний li, если `step.equipment` пустой
\+ пропали поварешки перед названиями(

Reply:
Добавлено и исправлено